src/app/components/bio-summary-card/bio-summary-card.component.ts

```diff
@@ -69,12 +69,16 @@ import { AppRoutes } from '../../app.routes';
 	imports: [CommonModule, NgOptimizedImage, NgIf, ResourceComponent, PortableTextParserComponent, RouterLink],
 })
 export class BioSummaryCardComponent implements OnInit {
-	@Input({ required: true }) story!: Story;
+	story = input<Story>();
```

Comment:
Since this input previously was marked with `{required: true}` we need to define the input signal with `input.required()`